ChangeLog

```diff
@@ -1,3 +1,43 @@
+# 2024-07-31 [version 1.14.0]
+## 🐣 New Features
+* Upgrade Java version to 8.
+* Provide shutdown(long, TimeUnit) method for ArcusClient.
+* Bulk methods using varargs keys are deprecated. Instead, methods using collection keys are still available.
+* Add auto import setting.
+* Add DNS cache TTL verification.
+* Add ArcusClient factory methods without ConnectionFactoryBuilder.
+* ArcusClient factory methods without zookeeper address are deprecated.
+* Add async mop upsert method.
+
+## 🔧 Enhancements
+* Refactor redundant concurrent data structure and unnecessary modifier.
+* TranscoderService is removed. Decoding value by asyncGet/asyncGetBulk will be done in user thread (like tomcat thread). BulkService properties in ConnectionFactoryBuilder is not available.
```

Comment:
* TranscoderService field in MemcachedClient and BulkService properties in ConnectionFactoryBuilder are removed. Decoding value with transcoder will be done in user thread, such as tomcat thread.